src/FSharpPlus/Converter.fs

```diff
@@ -106,28 +123,35 @@ type TryParse with
         let mutable r = Unchecked.defaultof< ^R>
         if (^R: (static member TryParse : _ * _ -> _) (x, &r)) then Some r else None
 
+    #if !FABLE_COMPILER
     static member inline TryParse (_: ^t when ^t: null and ^t: struct, _: Default1) = id
     static member inline TryParse (_: 'R, _: Default1) = fun x -> (^R: (static member TryParse : string -> 'R option) x)
-
+    #endif
 
 type Parse =
     inherit Default1
     static member inline Parse (_: ^R                  , _: Default1) = fun (x:string) -> (^R: (static member Parse : _ -> ^R) x)
     static member inline Parse (_: ^R                  , _: Parse   ) = fun (x:string) -> (^R: (static member Parse : _ * _ -> ^R) (x, CultureInfo.InvariantCulture))
 #if NET35
 #else
+#if !FABLE_COMPILER
     static member        Parse (_: 'T when 'T : enum<_>, _: Parse   ) = fun x ->
         (match Enum.TryParse (x) with
             | (true, v) -> v
             | _         -> invalidArg "value" ("Requested value '" + x + "' was not found.")
         ) : 'enum
 #endif
+#endif
```

Comment:
double #endif?

Reply:
If you see there are two compiler directives:
`#if NET35 ... #else ... #endif` and `#if !FABLE_COMPILER ... #endif`
The double `#endif` is for each `#if`.